teuthology/task/kernel.py

```diff
@@ -1021,11 +1049,35 @@ def task(ctx, config):
             if need_to_install_distro(ctx, role):
                 need_install[role] = 'distro'
                 need_version[role] = 'distro'
+        elif role_config.get("brew", None):
+            # installing a kernel from brewhub
+            build_id = role_config.get("brew")
+            if system_type != "rhel":
+                msg = (
+                    "Installing a kernel from brew is only supported "
+                    "on rhel systems. System type is {system_type}."
+                )
+                msg = msg.format(system_type=system_type)
+                log.error(msg)
+                ctx.summary["failure_reason"] = msg
+                ctx.summary["status"] = "dead"
+                raise ConfigError(msg)
+
+            # FIXME: this install should probably happen somewhere else
+            # but I'm not sure where, so we'll leave it here for now.
+            install_package('koji', role_remote)
```

Comment:
I thought about putting this call in packaging.get_brew_build_info.

Would that be terrible?